minikube/bootstrap.sh

```diff
@@ -91,6 +102,79 @@ upgradeHomebrewPackages() {
   done
 }
 
+# upgradeChocoPackages upgrades required Chocolatey packages to latest version.
+upgradeChocoPackages() {
+  runIfNot "choco list --local-only | grep 'minikube'" \
+    choco install minikube
+  if [ ! $(choco outdated | grep 'minikube') ]; then
+    runCmd \
+      choco upgrade minikube
+  fi
+  for pkg in kubernetes-cli kubernetes-helm; do
+    if [ ! $(choco list --local-only | grep $pkg | awk '{ print $1}') ]; then
+      runCmd \
+        choco install $pkg
+      echo "install $pkg"
```

Comment:
@targs08 there is no need in this `echo`. The `runCmd` function will print the command.